src/main/java/com/google/api/codegen/transformer/java/JavaGapicSurfaceTestTransformer.java

```diff
@@ -396,7 +437,7 @@ private InitCodeContext createResponseInitCodeContext(
     if (context.getMethodConfig().isPageStreaming()) {
       // Initialize one resource element if it is page-streaming.
       PageStreamingConfig config = context.getMethodConfig().getPageStreaming();
-      String resourceFieldName = config.getResourcesField().getSimpleName();
+      String resourceFieldName = config.getResourcesFieldConfig().getField().getSimpleName();
```

Comment:
maybe add a `getResourcesFieldName()` method to PageStreamingConfig to make this simpler (I have seen this in a couple places now)